### ISSUE TITLE
Sheyla: Debugging, 4 weeks

### ISSUE THREAD

**Issue by sheybusta** [HackYourFutureBelgium/class-13-14#221]
> [module repository](https://github.com/HackYourFutureBelgium/debugging)

### Week 1
- [x] ./stepping-through
- [ ]  ./describing-errors: until it's almost easy (going through)
- [ ]  ./interact: through conditionals
- [ ]  ./isolate: through conditionals

#### I Need Help:

To understand errors. Sometimes takes me a lot of time to find what is the error about and following step by step the whole idea of debugging is hard. Nevertheless, I think that will come with practice. 

#### What went well?

I've been studying together with Alina in some examples of the module. We try to help each other but sometimes we are complete lost. 

#### What went less well?

Following step by step whole process of debudding. 

#### Lessons Learned

I need to study more and more. 

### Sunday Prep Work
- Type Conversion
  - [javascript.info](https://javascript.info/type-conversions)
  - [./isolate](./isolate): explicit coercion
- Loops
  - [javascript.info](https://javascript.info/while-for)
  - [Mosh](https://www.youtube.com/watch?v=s9wW2PpJsmQ)
  - [./isolate](./isolate): while loops and for loops
- [./interact](./interact): numbers (examples)

**Comment by colevandersWands** [comment]
> We try to help each other but sometimes we are complete lost.

Let's hear it on Slack when you're lost!  I'm sure you're not the only ones blocked by the same question.

> Nevertheless, I think that will come with practice.

200%.  stepping through and debugging code is hard, that's why this module is 4 weeks so no rush. If you keep at it you'll get there.  And you don't need to master it in these 4 weeks either, you'll be debugging from now until the last day you ever code.  Plenty of time to practice : )

**Comment by sheybusta** [comment]
Indeed! I will try to be more patient with myself. I came across how important its to read carefully and going through line per line in Js.

**Comment by colevandersWands** [comment]
👋  @sheybusta, `Probably I need more help with loops.` - got any questions in particular?  Let me know what material has been the most helpful with loops so far and I can put together more like it.

PS.  Next week can you post your check-in as a comment in your issue?  It makes it easier to follow your progress, like in #217
